### PR TITLE
client

### PR DESCRIPTION
**Pull request by kashike**
[API](https://github.com/SpongePowered/SpongeAPI/pull/1804) | [**Common**](https://github.com/SpongePowered/SpongeCommon/pull/1868) | [Forge](https://github.com/SpongePowered/SpongeForge/pull/2096)

### CODE REVIEW

File: src/main/java/org/spongepowered/common/mixin/core/network/rcon/MixinRConThreadClient.java [hunk 1]
```diff
@@ -87,7 +88,7 @@ public InetSocketAddress getVirtualHost() {
     @Inject(method = "closeSocket", at = @At("HEAD"))
     public void rconLogoutCallback(CallbackInfo ci) {
         if (this.loggedIn) {
-            SpongeImpl.getScheduler().callSync(() -> {
+            ((SpongeScheduler) Sponge.getServer().getScheduler()).callSync(() -> {
```
Comment (gabizou) [inline src/main/java/org/spongepowered/common/mixin/core/network/rcon/MixinRConThreadClient.java]: Make this a method already..... skip the casting....

Comment (kashike) [reply src/main/java/org/spongepowered/common/mixin/core/network/rcon/MixinRConThreadClient.java]: This was before some refactoring.

**Comment by Zidane** [comment]
Closing as this has already been, somewhat, implemented in api-8.

**Comment by Dockter** [comment]
Yes, but what about API-7?

**Comment by Zidane** [comment]
Not happening on 7,

**Pull request closed without merging**